Author: Andreas Tille <tille@debian.org>
Date: Wed, 22 May 2013 13:27:40 +0200
Description: When building with --format-security (Debhelper 9 hardening)
 this patch is needed to build successfully

--- a/src/locfit/makecmd.c
+++ b/src/locfit/makecmd.c
@@ -200,7 +200,7 @@ char *cmdline;
   /* vs is used to store the command line string. */
   sprintf(vn,"=clstr%d",clcount);
   vs = createvar(vn,STSYSTEM,1+strlen(cmdline),VCHAR);
-  sprintf((char *)vdptr(vs),cmdline);
+  sprintf((char *)vdptr(vs),"%s",cmdline);
 
   /* va is used to store pointers to the command line fields. */
   sprintf(vn,"=cline%d",clcount);
